sympy/concrete/summations.py

```diff
@@ -253,6 +258,157 @@ def _eval_simplify(self, ratio, measure):
+        References
+        ==========
+
+        .. [1] https://en.wikipedia.org/wiki/Convergence_tests
+
+        Examples
+        ========
+
+        >>> from sympy import Interval, factorial, S, Sum, Symbol, oo
+        >>> n = Symbol('n', integer=True)
+        >>> Sum(n/(n - 1), (n, 4, 7)).is_convergent()
+        True
+        >>> Sum(n/(2*n + 1), (n, 1, oo)).is_convergent()
+        False
+        >>> Sum(factorial(n)/5**n, (n, 1, oo)).is_convergent()
+        False
+        >>> Sum(1/n**(S(6)/5), (n, 1, oo)).is_convergent()
+        True
+        """
+        from sympy import Interval, Integral, singularities, symbols
+        from sympy.solvers.solveset import solveset
+        p, q = symbols('p q', cls=Wild)
+
+        sequence_term = self.function
+        sym = self.limits[0][0]
+        lower_limit = self.limits[0][1]
+        upper_limit = self.limits[0][2]
+        interval = Interval(lower_limit, upper_limit)
+
+        if lower_limit.is_finite and upper_limit.is_finite:
+            return True
+
+        # transform sym -> -sym and swap the upper_limit = oo and lower_limit = - upper_limit
+        if lower_limit is -oo:
+            sequence_term = sequence_term.xreplace({sym: -sym})
+            lower_limit = -upper_limit
+            upper_limit = oo
+
+        ###  -------- Divergence test ----------- ###
+        try:
+            if limit(sequence_term, sym, upper_limit) != S.Zero:
+                return False
+        except NotImplementedError:
+            pass
+
+        ### --------- p-series test (1/n**p) ---------- ###
+        p1_series_test = sequence_term.match(sym**p)
+        p2_series_test = sequence_term.match((1/sym)**p)
+        if not p1_series_test is None:
+            if p1_series_test[p] < -1:
+                return True
+            if p1_series_test[p] > -1:
+                return False
+
+        if not p2_series_test is None:
+            if p2_series_test[p] > 1:
+                return True
+            if p2_series_test[p] < 1:
+                return False
+
+        ### ----------- root test ---------------- ###
+        lim = limit(abs(sequence_term)**(1/sym), sym, oo)
+        if lim < 1:
+            return True
+        if lim > 1:
+            return False
+
+        ### ------------- alternating series test ----------- ###
+        d = symbols('d', cls=Dummy)
+        dict_val = sequence_term.match((-1)**p*q)
+        if dict_val[p] == sym or dict_val[p].has(sym):
+            if solveset((dict_val[q].subs({sym: d + 1})/dict_val[q].subs({sym: d})) < 1, d, interval) == interval and \
+                limit(dict_val[q], sym, oo).is_finite:
+                    return True
+            return False
+
+        ### ------------- comparison test ------------- ###
+        order = O(sequence_term, (sym, oo))
+        if order.contains(O(1/sym, (sym, oo))):
+            return False
+
+        # (1/(n*log(n)**p)) comparison
+        log_n_test = sequence_term.match((1/(sym*(log(sym))**p)))
```

Comment:
I have no experience with matching, but I suspect that this should be more general to take care of terms like ``2/(n*log(n)**2)``. (Numerator different from one.)

Besides, this only works in a very special case and is more suitable for integral test. A comparison would be needed e.g. if the denominator would have an additional term of lower order (or growth), but that is harder to implement. There could be (integral) tests for these "log" cases.